### PR TITLE
Update to Erlang/OTP 24

### PR DESCRIPTION
**Pull request by vbaranov**
## Motivation

Update to Erlang/OTP 24

## Changelog

- Erlang/OTP 24 pinned
- bitwalker/alpine-elixir-phoenix:1.12.2 -> bitwalker/alpine-elixir-phoenix:1.12.3

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [x] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [x] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [x] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM bitwalker/alpine-elixir-phoenix:1.12.2
+FROM bitwalker/alpine-elixir-phoenix:1.12.3
```
Comment (sidhujag) [inline docker/Dockerfile:1]: https://github.com/bitwalker/alpine-elixir-phoenix/blob/master/VERSION or https://hub.docker.com/r/bitwalker/alpine-elixir-phoenix/tags?page=1&ordering=last_updated

Does 1.12.3 exist? you can use sidhujag/ domain if need be

Comment (sidhujag) [reply docker/Dockerfile:1]: What do you mean @zalam003 it does exist under sidhujag/

Comment (zalam003) [reply docker/Dockerfile:1]: @sidhujag If you search Docker hub, you will see `alpine-elixir-phoenix` latest is 1.12.2:
https://hub.docker.com/r/bitwalker/alpine-elixir-phoenix/tags?page=1&ordering=last_updated&name=1.12.

Comment (sidhujag) [reply docker/Dockerfile:1]: If you search on docker hub under sidhujag you will see 1.12.3 is the latest version, please re-read what I said.

Comment (zalam003) [reply docker/Dockerfile:1]: Not really arguing with you @sidhujag.  My point was the current version of Dockerfile will not work.  We need to either use:
```
FROM bitwalker/alpine-elixir-phoenix:1.12.2
```
or
```
FROM sidhujag/alpine-elixir-phoenix:1.12.3
```

My recommendation is to use an official version of the container and not use any personal ones.  Also we should also not move between 

FYI @vbaranov

Comment (sidhujag) [reply docker/Dockerfile:1]: @zalam003 its as official as bitwalker's is. If you go into the github you can see it points to bitwalker's elixir repo for 1.12.3, he has not simply updated the links to point upstream which is the only difference in mine.

**Pull request merged**